patches/evil-dns+0.2.0.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/node_modules/evil-dns/evil-dns.js b/node_modules/evil-dns/evil-dns.js
+index 740f40d..83986c7 100644
+--- a/node_modules/evil-dns/evil-dns.js
++++ b/node_modules/evil-dns/evil-dns.js
+@@ -25,10 +25,12 @@ dns.lookup = function(domain, options, callback) {
+ 
+ 	for (i = 0; i < domains.length; i++) {
+ 		var entry = domains[i];
+-		if (domain.match(entry.domain)) {
+-			if (!family || family === entry.family) {
+-				return callback(null, entry.ip, entry.family);
+-			}			
++		if (domain.match(entry.domain) && (!family || family === entry.family)) {
```

Comment:
is it worth adding a comment in the patch as to why this is necessary?

Reply:
Added: [`56524b7` (#30394)](https://github.com/cypress-io/cypress/pull/30394/commits/56524b7de45f13ec82da008e1814e414250ff684)